packages/snap-preact-components/src/components/Templates/Search/Search.tsx

```diff
@@ -329,8 +436,15 @@ export interface SearchProps extends ComponentProps {
 	hideBottomToolBar?: boolean;
 	hideMerchandisingBanners?: boolean | string[];
 	toggleSidebarButtonText?: string;
+	hideLayoutSelector?: boolean;
+	layoutConfig?: layoutConfig;
 }
 
+export type layoutConfig = {
+	options: layoutOption[];
+	default?: layoutOption;
+};
+
```

Comment:
Repeating an option to make it default is repetitive - we should come up with a better way of setting the default.